ICE/main.cpp

```diff
@@ -57,8 +70,9 @@ int main(int argc, char **argv){
 
     ICE *ice;
     ice = new ICE();
-    ice->LoadEdgeList(network_file);
+    ice->LoadEdgeList(text_file);
     ice->Init(dimensions);
+    ice->getVocabCount();
     
```

Comment:
so now we change the default version to "train stage 1 and then to train stage 2" ? or we'll have an option to do this?